lua52/PSPBUILD

```diff
@@ -0,0 +1,39 @@
+pkgname=lua52
+pkgver=5.2.4
+pkgrel=1
+pkgdesc="a powerful, efficient, lightweight, embeddable scripting language (version 5.2)"
+arch=('mips')
+url="https://lua.org"
+license=('MIT')
+groups=('pspdev-default')
```

Comment:
This should probably be moved to lua54 if we want to make that the default.

Reply:
Still needs to be tested how it behaves with several conflicting packages.

Reply:
I'll merge this to my repo's master, then we can do a test.

Reply:
Here is the GitHub Action: https://github.com/sharkwouter/psp-packages/actions/runs/9662990777

Once it's complete, you can change the server in `$PSPDEV/etc/pacman.conf` to `https://sharkwouter.github.io/psp-packages/` for testing.